tricat_211/CircleObstacleDWA[] circles
